src/appMain/CMakeLists.txt

```diff
@@ -138,6 +150,20 @@ if(ENABLE_LOG)
   list(APPEND LIBRARIES expat -L${EXPAT_LIBS_DIRECTORY})
 endif()
 
+include_directories (
+  ${default_media_inc}
+)
+
+set (SOURCES
+  main.cc
+  life_cycle.cc
+  signal_handlers.cc
+)
+
+if( NOT CMAKE_BUILD_TYPE )
```

Comment:
@LuxoftAKutsan yes, it already exists in main CMaleLists.txt, removed in https://github.com/LuxoftAKutsan/sdl_core/commit/69b0106c66b759cb7e1c5b56cd36e7c35b1a4867